Gems/DiffuseProbeGrid/Code/Source/Render/DiffuseProbeGridQueryFullscreenPass.h

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#include <Atom/RPI.Public/Pass/RenderPass.h>
+#include <Atom/RPI.Public/Shader/Shader.h>
+#include <Atom/RPI.Public/Shader/ShaderResourceGroup.h>
+#include <Render/DiffuseProbeGrid.h>
+#include <Render/DiffuseProbeGridFeatureProcessor.h>
```

Comment:
nit: it looks like Shader, and ShaderSeourceGroupLayout, can be forward declared, and I don't see any usage of DiffuseProbeGridFeatureProcessor in here.